pybase/client.py

```diff
@@ -382,7 +387,6 @@ def _create_new_region(self, response, table):
                 # hosted on.
                 server_loc = cell.value
                 host, port = cell.value.split(b':')
-                host = host.decode("utf-8")
```

Comment:
this now happens in the `Client` class init in `pybase/region/client.py`

Reply:
If you expand the code below, we use the host and port to create a new client
`new_client = region.NewClient(host, port, self.pool_size, secondary=self.secondary)`
which is why Ian originally added the bytes -> unicode conversion here since we need the host to be in unicode

However, since I moved that handling into the `Client()` init (https://github.com/Flipboard/PyBase/pull/13/files#r988402638) we don't need to do this here

does that make sense?